README.md

```diff
@@ -1 +1,5 @@
 # testingSquash
+
+some content
```

Comment:
this is great

Reply:
more! https://github.com/cwsmith/testingSquash/pull/1/commits/d8e408c2f7996883e51df7c9fe093243fc61fada